### ISSUE TITLE
Добавить requirements для разработки

### ISSUE THREAD

**Issue by vrkirillov** [newsviz/Spiders#8]

**Comment by iggisv9t** [comment]
Только пожалуйста не через pip freeze, и где можно через `>=` а не `==`

**Comment by vrkirillov** [comment]
Допустимо без указания версии для пакетов для разработки (black, isort)?

**Comment by iggisv9t** [comment]
Да, конечно. Я про случаи, когда ставишь чью-то штуковину, а она тебе начинает окружение даунгрейдить.
